src/lib/webpack/run-webpack.js

```diff
@@ -38,7 +38,12 @@ const devBuild = async (env, onprogress) => {
 				let serverAddr = `${protocol}://${host}:${chalk.bold(port)}`;
 				let localIpAddr = `${protocol}://${ip.address()}:${chalk.bold(port)}`;
 
-				process.stdout.write(chalk.green('\nCompiled successfully!!\n\n'));
+				if (stats.hasErrors()) {
+					process.stdout.write(chalk.red('\Build failed!\n\n'));
+				} else {
+					process.stdout.write(chalk.green('\nCompiled successfully!\n\n'));
+				}
```

Comment:
I'm good with that format.  I believe the "compiled successfully" bit can be done as the completion message of ProgressBarPlugin if it isn't already.